google/auth/credentials.py

```diff
@@ -56,7 +56,9 @@ def expired(self):
         Note that credentials can be invalid but not expired becaue Credentials
         with :attr:`expiry` set to None is considered to never expire.
         """
-        now = _helpers.utcnow()
+        # Err on the side of reporting expiration early so that we avoid
+        # the 403-refresh-retry loop.
+        now = _helpers.utcnow() - _helpers.CLOCK_SKEW
```

Comment:
Calling this variable `now` is a lie. You'd rather check `self.expiry + _helpers.CLOCK_SKEW <= now` for clarity?